zilencer/management/commands/populate_db.py

```diff
@@ -522,6 +556,8 @@ def handle(self, **options: Any) -> None:
         personals_pairs = [random.sample(user_profiles_ids, 2)
                            for i in range(options["num_personals"])]
 
+        create_alert_words(zulip_realm.id)
```

Comment:
We could do this inside an `if not options['test_suite']` if we just want it in the development environment, but probably we do want them present in unit tests too.

Reply:
yeah, I think it's good to have in the test data